### DISCUSSION TITLE
ERROR install @phosphor-icons/react

### DISCUSSION THREAD

**Discussion by diegolauar** [phosphor-icons/homepage#486] [Show and tell]
If you get this error: Failed to resolve entry for package "@phosphor-icons/react". The package may have incorrect main/module/exports specified in its package.json.

Try install: npm install phosphor-react

Use: import { . . . } from "phosphor-react"

**Comment by rektdeckard** [comment]
Moved to https://github.com/phosphor-icons/react/issues/96. Please refrain from spamming duplicates of the same issue on the different repositories of this project.
